### ISSUE TITLE
Touch support

### ISSUE THREAD

**Issue by angelwzr** [ventoy/Ventoy#151]
First of all, thanks for creating and maintaining such a useful tool. I like it a lot and use it very often.

My issue/suggestion is touch support for Ventoy UI. 
I have to deal with OS reinstall on tablets and other touch devices without keyboards a lot and having Ventoy with touch support would be a total time saver.

One possible (at least I think it is) scenario for that is adding support (maybe as a plugin?) for alternative bootloader instead of grub, like rEFInd, which supports touch input after simply enabling it in config.

Anyway, I'm pretty happy with Ventoy so keep up the great work! :)

**Comment by ventoy** [comment]
You only need UEFI for touch support?  
Maybe Ventoy can integrate the related efi driver in rEFInd.

**Comment by angelwzr** [comment]
@ventoy yeah, touch support in UEFI mode would be enough, no need for legacy :)

**Comment by ventoy** [comment]
Can you find a way to know which efi driver works?

**Comment by angelwzr** [comment]
Can't seem to find anything mentioned on the driver part in the actual code of rEFInd, only that it's handled in pointer.h and pointer.c.

**Comment by sharicov** [comment]
Please add touch support, have tablet without keyboard.

**Comment by meydominic** [comment]
Touch support would be really nice.

Maybe you can add navigating with volume keys as alternative.

On my dell tablet the vol-up key ist to navigate through menu and the vol-down key is to select the option. And the windows button ist for getting back.

Best regards,
Dominic

**Comment by hexman1999** [comment]
Please add this feature it is necessary for tablet user as ventoy is a (universal multiboot solution) for all devices 


No need to make it act as a mouse

You can make it act like basic and most needed keyboard shortcuts to boot 

Swipe from bottom to top -> arrow up
Swipe from up to down -> arrow down
Swipe from right to left -> enter
Swipe from left to right -> esc
